From: Simon McVittie <smcv@debian.org>
Date: Thu, 19 Jan 2017 11:23:30 +0000
Subject: Sourced test snippets: remove shebang and make non-executable

They are installed non-executable, which makes Debian's Lintian
packaging consistency check complain that #! is only useful
in executable scripts. But in fact they are not useful to execute
directly (they rely on setup being done in the script that sources
them), so just chmod them -x.

Forwarded: https://github.com/ostreedev/ostree/pull/652
---
 tests/admin-test.sh   | 3 ++-
 tests/archive-test.sh | 4 ++--
 tests/basic-test.sh   | 4 ++--
 tests/pull-test.sh    | 4 ++--
 4 files changed, 8 insertions(+), 7 deletions(-)
 mode change 100755 => 100644 tests/admin-test.sh
 mode change 100755 => 100644 tests/archive-test.sh
 mode change 100755 => 100644 tests/basic-test.sh
 mode change 100755 => 100644 tests/pull-test.sh

diff --git a/tests/admin-test.sh b/tests/admin-test.sh
old mode 100755
new mode 100644
index 76fc8b85..cc06fe6f
--- a/tests/admin-test.sh
+++ b/tests/admin-test.sh
@@ -1,4 +1,5 @@
-#!/bin/bash
+# This file is to be sourced, not executed
+
 # Copyright (C) 2011,2014 Colin Walters <walters@verbum.org>
 #
 # This library is free software; you can redistribute it and/or
diff --git a/tests/archive-test.sh b/tests/archive-test.sh
old mode 100755
new mode 100644
index e6f67cf5..9c5f8a47
--- a/tests/archive-test.sh
+++ b/tests/archive-test.sh
@@ -1,5 +1,5 @@
-#!/bin/bash
-#
+# This file is to be sourced, not executed
+
 # Copyright (C) 2011 Colin Walters <walters@verbum.org>
 #
 # This library is free software; you can redistribute it and/or
diff --git a/tests/basic-test.sh b/tests/basic-test.sh
old mode 100755
new mode 100644
index bb387501..f3e37239
--- a/tests/basic-test.sh
+++ b/tests/basic-test.sh
@@ -1,5 +1,5 @@
-#!/bin/bash
-#
+# This file is to be sourced, not executed
+
 # Copyright (C) 2011 Colin Walters <walters@verbum.org>
 #
 # This library is free software; you can redistribute it and/or
diff --git a/tests/pull-test.sh b/tests/pull-test.sh
old mode 100755
new mode 100644
index 56258b6c..56b24a0c
--- a/tests/pull-test.sh
+++ b/tests/pull-test.sh
@@ -1,5 +1,5 @@
-#!/bin/bash
-#
+# This file is to be sourced, not executed
+
 # Copyright (C) 2011 Colin Walters <walters@verbum.org>
 #
 # This library is free software; you can redistribute it and/or
